### DISCUSSION TITLE
Compiti senza entrare in agenda

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#584] [Idee e proposte]
Salve, sarebbe possibile inserire la possibilità di firmare e inserire argomenti e compiti direttamente dalla firma con data da consegnare come in argo (allego foto) 
![sddefault](https://github.com/iisgiua/giuaschool/assets/137722020/174a73a1-83fc-438e-9802-fb6a67887c73)

**Comment by trinko** [comment]
Al momento non sono previste modifiche all'interfaccia usata per richiamare le funzionalità dell'applicazione.

Aggiungo che personalmente sono dell'opinione che sia più chiaro e semplice tenere separate le due funzionalità: cercare di mettere tutto insieme finisce per rendere più complicata la comprensione delle operazioni da svolgere.
